client/cmd/rollback.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/piplabs/story/client/app"
+	storycfg "github.com/piplabs/story/client/config"
+	libcmd "github.com/piplabs/story/lib/cmd"
+	"github.com/piplabs/story/lib/log"
+)
+
+type rollbackConfig struct {
+	RemoveBlock     bool // See cosmos-sdk/server/rollback.go
+	RollbackEVM     bool
+	RollbackHeights int64
```

Comment:
Why don't we use type `uint64` for RollbackHeights? And I think it would be good to validate this config before rolling back if this is greater than 0.

Reply:
reflected in e78cb90